packages/react-server/core/util/PageUtil.js

```diff
@@ -17,6 +17,85 @@ var {isTheFold, markTheFold} = require('../components/TheFold');
 // These three data structure define the page interface.
 
 
+var PageConfig = (function(){
```

Comment:
OK. I was unclear if those comments were for clients of this file (in which case they make more sense to me at the top of the file) or folks who are going to muck with the impl of this file (in which I case they definitely belong below). I'll move them!